Integrations/src/main/java/io/deephaven/integrations/learn/Computer.java

```diff
@@ -67,6 +70,10 @@ public Computer(Table table, Function<Object[], Object> modelFunc, Input[] input
      */
     public boolean clear() {
 
+        for (FutureOffset futureOffset : foSet) {
+            futureOffset.clear();
+        }
+        foSet.clear();
```

Comment:
```suggestion
        // clear FutureOffsets so that objects can be garbage collected.
        for (FutureOffset futureOffset : foSet) {
            futureOffset.clear();
        }
        
        foSet.clear();
```